app/services/subscriptions/validate_service.rb

```diff
@@ -35,12 +36,48 @@ def valid_plan?
     end
 
     def valid_subscription_at?
-      return true if args[:subscription_at].respond_to?(:strftime)
-      return true if args[:subscription_at].is_a?(String) && DateTime._strptime(args[:subscription_at]).present?
+      return true if is_valid_format?(args[:subscription_at])
 
       add_error(field: :subscription_at, error_code: 'invalid_date')
 
       false
     end
+
+    def valid_ending_at?
+      return true if args[:ending_at].blank?
+
+      if is_valid_format?(args[:ending_at]) &&
+        is_valid_format?(args[:subscription_at]) &&
+        ending_at.to_date > Time.current.to_date &&
+        ending_at.to_date > subscription_at.to_date
```

Comment:
Layout/MultilineOperationIndentation: Align the operands of a condition in an `if` statement spanning multiple lines.